jdaviz/configs/specviz2d/helper.py

```diff
@@ -142,6 +143,11 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
         show_in_viewer : bool
             Show data in viewer(s).
 
+        ext : int, optional
```

Comment:
The way HDUList is accessed, I think specs like `ext='SCI'` or `ext=('SCI', 1)` will also work though I guess we don't have to advertise it? 🤷